packages/server-wallet/e2e-testing/test-data/sample-direct-load-data.json

```diff
@@ -1,39 +1,3688 @@
 [
  {
   "type": "CreateChannel",
-  "jobId": "better-massive-shaggy-battery",
```

Comment:
I think it's pretty nasty to check in these (large) files that randomly change. I don't really see the benefit of having example load files checked in.

Consider:
- I change the load generator/runner in some way
- I forget to update the sample files
Then, when you checkout my changes and try to run the sample files, they won't work!